db/migrate/20190619153428_add_subscription_status_to_follows.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubscriptionStatusToFollows < ActiveRecord::Migration[5.2]
+  def change
+    add_column :follows, :subscription_status, :string, default: "all_posts", null: false
```

Comment:
We could change the default to `all_articles` if we want to keep it consistent.

Reply:
+1 for consistency. I think `subscription_status` also needs a validator in the model to check it belongs to a list of available values.